qiskit_experiments/framework/json.py

```diff
@@ -477,6 +477,14 @@ def default(self, obj: Any) -> Any:  # pylint: disable=arguments-differ
                     "version": get_object_version(cls),
                 },
             }
+        if isinstance(obj, Gate):
```

Comment:
Is there a reason this is `Gate` instead of `Instruction` like in the runtime encoder?
https://github.com/Qiskit/qiskit-ibm-runtime/blob/adfd6f2c1f796ffed4c4d9ee3f2beea486a32acf/qiskit_ibm_runtime/utils/json.py#L215-L224

Reply:
No reason. See here: [5c93061](https://github.com/Qiskit/qiskit-experiments/pull/684/commits/5c93061706c68ecdbc87c0767597002192df46c1)